index.js

```diff
@@ -44,13 +44,13 @@ for (let i = 0; i < datasets.length; i++) {
     var card =  `<div class="card" id="dataset-${i}">` +
                     `<div class="card-header" id="heading-${i}">` +
                         '<h6 class="mb-0">' + 
-                            `<a class="collapsed text-left stretched-link" type="button" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
+                            `<a class="collapsed text-left stretched-link" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
                                 `${title}` + 
                             '</a>' +
                         '</h6>' +
                         '<div class="row justify-content-end">' +
-                            `<button id="showBounds-${i}" class="btn btn-link" onclick="toggleBounds(${i});" style="display: none; z-index: 2000;">Show Bounds</button>` +
-                            `<button id="hideBounds-${i}" class="btn btn-link" onclick="toggleBounds(${i});" style="z-index: 2000;">Hide Bounds</button>` +                        
+                            `<button id="showBounds-${i}" class="btn btn-link" onclick="toggleBounds(${i});" style="display: none; z-index: 2000; font-size: 50%;">[Show Bounds]</button>` +
```

Comment:
![grafik](https://user-images.githubusercontent.com/15731649/56463917-24891880-63ac-11e9-9f29-2206c764b4c4.png)
I think that's too small, I can barely read it. Seems like I either need a screen with more DPI or new glasses... :)

Reply:
Made larger for now. But eventually I'd like this replaced with some icon?

Reply:
Maybe the Location-marker icon? (Such as https://material.io/tools/icons/?icon=location_on&style=baseline and https://material.io/tools/icons/?icon=location_off&style=baseline)

Reply:
👍 

Very nice.